Templates/Swift/Pay/Includes/Enum.stencil

```diff
@@ -0,0 +1,17 @@
+{% if description %}
```

Comment:
The `Enum.stencil` in `Poi` is identical to this one, also there is also one in the `Common` folder, so this is probably not needed anymore?